packages/checkout/sdk/src/smartCheckout/routing/swap/quoteFetcher.ts

```diff
@@ -36,13 +37,15 @@ export const quoteFetcher = async (
 
     // Resolve all the quotes and link them back to the swappable token
     // The swappable token array is in the same position in the array as the quote in the promise array
-    const dexTransactionResponse = await Promise.all(dexTransactionResponsePromises);
+    const dexTransactionResponse = await Promise.allSettled(dexTransactionResponsePromises);
+
     dexTransactionResponse.forEach((response, index) => {
+      if (response.status === 'rejected') return; // Ignore any requests to dex that failed to resolve
```

Comment:
I agree with @ZacharyCouchman I think if there is an error that could be mitigated by the user and we couldn't find any swap options then responding with that might give them a pathway forward.. not sure what those error types would be though. But otherwise might just be too much noise as it is best effort.